src/test/java/com/transferzero/sdk/api/WebhooksApiTest.java

```diff
@@ -95,7 +97,10 @@ public void getWebhookEventsTest() throws ApiException {
     public void getWebhooksTest() throws ApiException {
         Integer page = null;
         Integer per = null;
-        WebhookDefinitionListResponse response = api.getWebhooks(page, per);
+        WebhookDefinitionListResponse response = api.getWebhooks()
+                .page(page)
+                .per(per)
```

Comment:
similarly, either make the examples useful by specifying details here, or just remove the calls altogether